### PR TITLE
fix: RootRelationalClassMapping compilation to include mapping filter

### LINKED ISSUE #175: Feature request: Add filter support for Relational Mapping
Support a filter lambda as part of relational mapping. This feature gives users the ability to filter through a lambda. The filter will be applied to the source which in this case is a relational mapping.

Example:

```
Class my::Person
{
   firstName : String[0..1];
   lastName : String[0..1];
}

###Relational
Database my::mainDb
(
   Table PersonTable(firstName VARCHAR(200), lastName VARCHAR(200))
   Filter PersonFilter(PersonTable.firstName = 'Utkarsh')
)

###Mapping
Mapping my::mainMap
(
   my::Person : Relational
   {
      ~filter [my::mainDb] PersonFilter
      firstName : [my::mainDb]PersonTable.firstName,
      lastName : [my::mainDb]PersonTable.lastName
   }
)
```

### PR DESCRIPTION
**Pull request by hardikmaheshwari**
| Q                       | A                                |
| ----------------------- | -------------------------------- |
| Fixed issues?           | Fixes #175  |
| Tests added + passed?   | Yes                              |
| Changeset added?        | Yes                              |
| Documentation PR link   |                                  |
| Any dependency changes? | No                               |

### CODE REVIEW

File: .changeset/red-otters-grin.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-studio': patch
+---
+
+fix rootRelationalClassMapping compilation to include filter
```
Comment (akphi) [inline .changeset/red-otters-grin.md:5]: Let's start with a capitalized word and end with a period for consistency

**Review by akphi — CHANGES_REQUESTED** [review]
Thanks @hardikmaheshwari! Just a minor changelog change

**Comment by akphi** [comment]
Also Im not too sure what happened to your CI, can you try to do a force push to retrigger it?

File: .changeset/red-otters-grin.md (hunk 1 above)
Comment (hardikmaheshwari) [reply .changeset/red-otters-grin.md]: Thanks done.

**Review by akphi — APPROVED** [review]

**Pull request merged**